src/frontend/apps/impress/src/features/service-worker/service-worker-api.ts

```diff
@@ -46,7 +46,7 @@ registerRoute(
 
```

Comment:
may I suggest this : 

```suggestion
const isDocumentApiUrl = (url: URL) =>
  isApiUrl(url.href) && /.*\/documents\/([a-z0-9-]+)\/$/g.test(url.href);

registerRoute(
  ({ url }) => isDocumentApiUrl(url),
```